apps/dsb-message-broker/src/app.module.ts

```diff
@@ -25,7 +25,8 @@ import { MessageModule } from './message/message.module';
                 CACHE_SERVER_URL: Joi.string().default('https://identitycache-dev.energyweb.org'),
                 WITH_SWAGGER: Joi.boolean().optional().default(true),
                 JWT_SECRET: Joi.string().required(),
-                PRIVATE_KEY: Joi.string().required()
+                PRIVATE_KEY: Joi.string().required(),
+                MB_DID: Joi.string().required().default('did:ethr:0x')
```

Comment:
MB_DID corresponds to the PRIVATE_KEY, I think it would also be great if we add validation if the PRIVATE_KEY is a owner or delegate of MB_DID.

Here the init function could be extended
https://github.com/energywebfoundation/dsb/blob/1333e39e1cfe1f46e781a5828190ff4bb340dec8/apps/dsb-message-broker/src/auth/auth.service.ts#L11

Reply:
now compares with init.did